code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -132,6 +132,10 @@ GLOBAL_LIST_INIT(protected_objects, list(/obj/structure/table, /obj/structure/ca
 	. = ..()
 	return . - creator
 
+/mob/living/simple_animal/hostile/mimic/copy/wabbajack_act()
+	src.visible_message(span_warning("[src] resists polymorphing into a new creature!"))
+	return
```

Comment:
```suggestion
	visible_message(span_warning("[src] resists polymorphing into a new creature!"))
```
* `src` is implied.
* empty `return` at the end of the proc is implied.
No need for the boilerplate.